compiler/luci-pass-value-py-test/test.lst

```diff
@@ -60,6 +60,7 @@ eval(Softmax_001 decompose_softmax)
 eval(Softmax_002 decompose_softmax)
 eval(UnidirectionalSequenceLSTM_003 unroll_unidirseqlstm)
 eval(UnidirectionalSequenceLSTM_004 unroll_unidirseqlstm)
+eval(Net_Shape_Add_000 fold_shape)
```

Comment:
can you mode this after `Net_Reshape_Reshape_000` ?
although there are some item not in alphabetical order, let's put them in order.
if you like, you can fire another PR to sort.

Reply:
> if you like, you can fire another PR to sort.

I think that it would be better!
If you don't mind, I'll sort the tests in alphabetical order in next PR 😄

Reply:
OK